docs/en/compute/Compute-HBase.md

```diff
@@ -69,39 +63,21 @@ Add the following properties to the same file `hbase-site.xml`.
 The last property is required to prevent HBase from flushing Alluxio file stream in a thread unsafe
```

Comment:
I think the first two `fs.alluxio.impl` and `fs.AbstractFileSystem.alluxio.impl` is in fact not required. can you double check?

Reply:
Removed the two properties and restart HBase in local computer, examples finish successfully.